static/app/components/emptyStateWarning.tsx

```diff
@@ -21,7 +21,7 @@ function EmptyStateWarning({small = false, withIcon = true, children, className}
     </EmptyMessage>
   ) : (
     <EmptyStreamWrapper data-test-id="empty-state" className={className}>
-      {withIcon && <IconSearch legacySize="54px" />}
+      {withIcon && <IconSearch size="xl" />}
```

Comment:
is this the same as 54px?

Reply:
so the sizes are not exact
`  xl: '32px',
  xxl: '72px'` is what's in the code, so I picked `xl` because it's the closest one and `xxl` felt way too big

Reply:
We could leave it the way it was. Making this change will affect other parts of the app and idk how they'll all look.